profiles/base.yaml

```diff
@@ -731,6 +731,9 @@ use_own_ingot_type:
 deed_own_ingot: false
 # spells that get woven in to various scripts, do not use cambrinth
 crafting_training_spells:
+# Toggle Sorcery during crafting or outdoorsmanship. Sorcery will always be blocked by forge.lic
+#ADVANCED OPTION MAY RESULT IN ARREST, USE WITH CAUTION!!
+cast_sorcery_training: false
```

Comment:
If this setting is intended to be used only in very specific circumstances (i.e. crafting or outdoorsmanship), what do you think about giving it a more specific name?